### PR TITLE
Revise README

### PR DESCRIPTION
**Pull request by ender503**
Explain more detail about this demonstration.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,38 @@
 # TangleID Demo Site
 
-This demo site is designed to illustrate the course site issue an claim to the participants.
+Tangle Demo Site is designed to illustrate how a certification website issues an claim to the identity holders.
 
-## Getting Started
+<img src="./images/overview.png" alt="Certification Demonstration Overview" style="width: 400px; display: block; margin-bottom: 20px"/>
+
+The scenario we choose is the certification process and the roles contain:
```
Comment (jserv) [inline README.md]: Insert the wording like...
> DIDs are useful for any application that benefits from self-administered, cryptographically verifiable identifiers such as personal identifiers, organizational identifiers, and identifiers for Internet of Things scenarios. Here...

File: README.md [hunk 2]
```diff
@@ -1,8 +1,38 @@
...
+ - Student: As a holder, a student is able to arrange his or her personal data and claims with verified carriers.
+ - College: As an issuer, a college can issue verifiable claims to the holders after verifying students' identity.
+
+In the following, we will provide some guidelines.
```
Comment (jserv) [inline README.md]: This statement is redundant.

File: README.md [hunk 3]
```diff
@@ -1,8 +1,38 @@
...
+
+At first, students (holders) have to create new digital identities on Tangle and the registration includes 2 steps:
+1. Download the application (APP) on devices, such as mobile or tablets
+2. Create a new account and sign in
+
+### Step2: Sign in by Verifying the Digital Identity
+
+<img src="./images/prove_identity.png" alt="Prove Own Control of Student Identity" style="width: 400px; display: block; margin-bottom: 20px"/>
+
+After the accounts created, the next step is Sign in. Students (Holders) are required to scan the QR Codes provided by the website (http://demokits.biilabs.io:4500/) in order to decide what data will be revealed.
```
Comment (jserv) [inline README.md]: Should be stick on port `4500`?

File: README.md (hunk 2 above)
Comment (ender503) [reply README.md]: Delete this line. 👌

**Pull request merged**